theatre/shared/src/notify.ts

```diff
@@ -47,7 +47,7 @@ const createHandler =
     }
 
     // @ts-ignore
-    return window[globalVariableNames.notifications]?.notify[type](...args)
+    return window?.[globalVariableNames.notifications]?.notify[type](...args)
```

Comment:
@vezwork our policy now is to treat all @theatre/core  usages of `window` as optional, right? I don't quite remember the work you did on keeping compatible with nextjs.